tests-examples/demo-todo-app.spec.ts

```diff
@@ -0,0 +1,437 @@
+import { test, expect, type Page } from '@playwright/test';
```

Comment:
Shall we remove this file and the directory because we already have an example test at `tests/example.spec.ts`?